src/components/Faucet/use-faucet.tsx

```diff
@@ -1,7 +1,18 @@
 import { useClient } from '@vocdoni/react-providers'
 
+export type signinUrlParams = {
+  param: string
+  value: any
+}
+
+export type authTypes = {
+  oauth?: number
+  open?: number
+  aragondao?: number
+}
+
 export const useFaucet = () => {
-  const { connected, signer, client } = useClient()
+  const { client } = useClient()
 
   // TODO: Remove this when the client is updated
```

Comment:
Since you're changing this file quite a lot, could you please make these changes? Theoretically, the latest SDK versions should not return v2 already, so it should be only remove the "url.replace" line below (but please test it :__))

Reply:
Oof, no... I'll do it. I've added this todo to the issue tracking the SDK upgrade: https://github.com/vocdoni/ui-scaffold/issues/399